### PR TITLE
Feat/jetson tx2

### PR DESCRIPTION
**Pull request by ltetrel**
I updated your code to make it work on a jetson-TX2 which is compatible with:
* [CUDA 10.2](https://docs.nvidia.com/cuda/archive/10.2/cuda-runtime-api/)
* [TensorRT 8.2.1](https://docs.nvidia.com/deeplearning/tensorrt/archives/tensorrt-821/)
* C++14

### CODE REVIEW

File: src/engine.cpp [hunk 1]
```diff
@@ -247,43 +238,42 @@ bool Engine::loadNetwork() {
...
+    //TODO: rollback libnvinfer API to TensorRT 8.2.1.9
+    //TODO: Needs a way to dynamically get input and output buffer size
+    int32_t nb_io_tensors = 2;
+    m_buffers.resize(nb_io_tensors);
+    // Tensor shape is (bs, chan, height, width)
+    nvinfer1::Dims4 inp_tensor_shape = {1, 3, 640, 640};
+    nvinfer1::Dims3 out_tensor_shape = {1, 84, 8400};
+    // Input and output tensor name
+    m_IOTensorNames.emplace_back("images");
+    m_IOTensorNames.emplace_back("output0");
```
Comment (cyrusbehr) [inline src/engine.cpp:250]: Can you look at v2.x of this project. Might give you some insight on how to dynamically get the input & output names and shapes, pre 8.6 

image

**Review by cyrusbehr — CHANGES_REQUESTED** [review]

File: src/engine.cpp (hunk 1 above)
Comment (ltetrel) [reply src/engine.cpp:250]: Thanks I will look into that, I was not sure actually how to get them dynamically as stated in the issue.
Another important issue is the INT8 compatibility.

Comment (cyrusbehr) [reply src/engine.cpp:250]: Ok. I'll merge in your changes for now so that others can reference it.

**Comment by cyrusbehr** [comment]
@all-contributors please add @ltetrel for code

**Pull request merged**